client/api/src/execution_extensions.rs

```diff
@@ -77,7 +77,7 @@ impl ExtensionsFactory for () {
 /// A producer of execution extensions for offchain calls.
 ///
 /// This crate aggregates extensions available for the offchain calls
-/// and is responsbile to produce a right `Extensions` object
+/// and is responsible to produce a right `Extensions` object
```

Comment:
Dumb question, do you mean "correct" object? I would use that unless this is referring to a right/left distinction.
```suggestion
/// and is responsible for producing a right `Extensions` object.
```

Reply:
Yes I think it means "correct" here.